hq/test/utils/attempt/AttemptValues.scala

```diff
@@ -42,5 +42,11 @@ trait AttemptValues extends Matchers {
         _ => false
       )
     }
+    def getFailedAttempt()(implicit ec: ExecutionContext): Option[FailedAttempt] = {
```

Comment:
Why would this return an `Option`? I think this should work more like the `value()` method above (which is in turn inspired by scalatest conventions like `EitherValues`. If you want a failed attempt then it's a test failure if you didn't get one (rather than a None that you then need to handle).

You're calling `.get` anyway so this way would also give much better test error messages.